hw/ip/kmac/lint/kmac.waiver

```diff
@@ -18,3 +18,7 @@ waive -rules {HIER_NET_NOT_READ NOT_READ INPUT_NOT_READ} \
 waive -rules {HIER_NET_NOT_READ NOT_READ INPUT_NOT_READ} \
     -location {kmac_staterd.sv} -regexp {'unused_.*} \
```

Comment:
can you check whether this waiver can be removed? the tool should already ignore unused signals with that prefix...

Reply:
Still seeing errors on `unused_*` signals.

```
HIER_NET_NOT_READ:   kmac_staterd.sv:46        Net 'unused_tlram_wdata' is not read from in module 
```

Reply:
FYI,those nets are connected to the submodule's outputs directly.

Reply:
Anyway, let me remove the line. I will check why `common.waiver` does not affect to this lint environment.

Reply:
Ok good, thanks!

Reply:
That's an interesting observation, I wasn't aware that nets directly connected to module outputs weren't caught by the existing rule.